packages/marketplace/src/selector/client.ts

```diff
@@ -43,3 +43,7 @@ export const selectIsWebComponentNegotiators = (state: ReduxState) => {
 export const selectAppDetail = (state: ReduxState) => {
   return state.appDetail
 }
+
+export const selectMyApps = (state: ReduxState) => {
+  return state.myApps
+}
```

Comment:
Could we use:
`export const selectMyApps = (state: ReduxState) => state.myApps`

Reply:
yes, but I prefer to use normal function like other selectors because we can more readable.